test/qunit/model/FHIRModel.integration.js

```diff
@@ -660,4 +663,26 @@ sap.ui.define([
 		this.oFhirModel.readLatestVersionOfResource(sPatientPath, fnSuccessCallback);
 	});
 
-});
\ No newline at end of file
+	QUnit.test("Test Submit Changes after version read", function(assert) {
+		var done = assert.async();
+		this.oFhirModel.create("Practitioner", {
+			name: [
+				{
+					family: "johnson1",
+					given: ["dexter1"]
+				}
+			],
+			birthDate: "1987-03-03"
+		}, "practitioner2");
+		var successFn = function (oData) {
+			this.oFhirModel.setProperty("/" + oData.resourceType + "/" + oData.id + "/birthDate", "1988-03-04");
+			delete this.oFhirModel.oData.Practitioner[oData.id].meta;
```

Comment:
I would add here the scenario which you've described in the PR also in the test code as a comment.

Reply:
done